File: sayhello.py
class Person:
  def __init__(self, altura, cor):
    self.altura = altura
    self.cor = cor

  def say_hello(self, name):
    print(f"Hello {name}! My height is {self.altura} and my color is {self.cor}.")

pricilla = Person(155, "white")
marcus = Person(176, "green")

pricilla.say_hello("Marcus")
marcus.say_hello("Hayley")

File: sayhello.js
class Person {
  constructor(altura, color) {
    this.altura = altura
    this.color = color
  }
  sayHello(name) {
    alert(`Hello ${name}! My height is ${this.altura} and my color is ${this.color}.`)
  }
}

const pricilla = new Person(155, "white")
const marcus = new Person(176, "green")

console.log(pricilla)
console.log(marcus)
